SU-SWS/stanford-jumpstart-deployer
# DEVOPS-156: Update module ds-7.x-2.15

# READY FOR REVIEW

# Summary
- This is related to the security advisory: https://www.drupal.org/sa-contrib-2018-019.

# Steps to Test

1. Pull the latest changes to linky_clicky-5.x, which should include an update to sites/uat/features/high-value/ds.feature.
2. Run ds.feature against a build of this branch.

# Affected Projects or Products
- Fairly limited sub-set of sites, as it is related to the submodule ds_extras.

# Associated Issues and/or People
- https://github.com/SU-SWS/linky_clicky/pull/195
- https://stanfordits.atlassian.net/browse/DEVOPS-156
